rom/scripts/GREEDY.md

```diff
@@ -0,0 +1,33 @@
+To use the greedy algorithm obtained from algorithm 2 of Choi et. al's
+paper "Gradient-based constrained optimization using a database
+of linear reduced-order models": https://arxiv.org/abs/1506.07849
+to build a ROM database with Laghos, use one of the following commands.
+
+Hyperreduction is not supported!
+
+To run locally without using srun:
+
+./scripts/laghos_greedy_alg_launcher.sh laghos ... (to run locally without using srun)
+
+To use sbatch, maintaining one node throughout the algorithm. sbatch configuration
+commands can be modified within the launcher script:
+
+sbatch scripts/laghos_greedy_alg_launcher.sh srun -n 1 -p pdebug laghos ...
+
+Example command with all available options:
+
+./scripts/laghos_greedy_alg_launcher.sh laghos -m data/cube01_hex.mesh -pt 211 -tf 0.01 -build-database -writesol -romsvds -greedy-param "bef" -greedy-param-min 0.5 -greedy-param-max 2.5 -greedy-param-size 5 -greedysubsize 2 -greedyconvsize 3 -greedytol 0.1 -greedyalpha 1.05 - greedymaxclamp 2.0 -greedysamptype random -greedyerrindtype useLastLifted
```

Comment:
Remove space before `greedymaxclamp`.

Reply:
When I run this, I get
`MFEM abort: The greedy algorithm procedure has completed!`
The message is positive, but the `abort` is negative. Is this good or bad? It looks like this is at the end of the offline run, so is this the intended way of ending?

Reply:
Yes, it's a bit tough because without being able to run Laghos simulations multiple times within one command, we have to run it multiple times in a script. The only way the script would know when to end would be either to check for some file/message, which is a potential option, or through the exit code (which is what the MFEM abort is for).

Reply:
I see, this is a tricky case for which the original Laghos driver was not designed. Using `MFEM_Abort` does seem convenient, but it can be confusing if one usually expects it to mean a failure. Also, the memory is not freed at the end, which I suppose doesn't really matter but is not good style. Maybe a cleaner way to end would be for `SaveROMDatabase` to return the exit code, so that the memory is still freed at the end of `main` and then the code is returned by `main` on a proper exit. Checking a file for a message is another possibility, but the return code seems simplest.